src/create_disk.sh

```diff
@@ -396,11 +397,34 @@ cat > $rootfs/.coreos-aleph-version.json << EOF
 }
 EOF
 
-install_uefi() {
-    # https://github.com/coreos/fedora-coreos-tracker/issues/510
-    # See also https://github.com/ostreedev/ostree/pull/1873#issuecomment-524439883
+bootloader_install_nons390x_bootupd_epoch_1() {
+    # For background on bootupd, see https://github.com/coreos/fedora-coreos-tracker/issues/510
+    # This requires bootupd 0.2.11 in the target system, and handles both BIOS and UEFI for x86_64 for example.
+    bootupd_args=()
+    if [ "${x86_bios_bootloader}" = 1 ]; then
+        bootupd_args+=("--device=$disk")
+    fi
+    chroot_run /usr/bin/bootupctl backend install --src-root="${deploy_root}" "${bootupd_args[@]}" "${rootfs}"
+    case "${arch}" in
+        x86_64|aarch64)
+            inject_grub_uefi
```

Comment:
optional: I think this would be slightly easier to follow if the `inject_grub_cfg` wasn't embedded in `inject_grub_uefi` but rather called at the same level:

```suggestion
            inject_grub_uefi
            inject_grub_cfg
```

i.e. it would make it clearer that x86_64 and aarch64 need BOTH `inject_grub_uefi` and `inject_grub_cfg`, but ppc64le only needs `inject_grub_cfg`.